### PR TITLE
.github: add workflow for system tests

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: .github/workflows/system-tests.yml [hunk 1]
```diff
@@ -0,0 +1,267 @@
+name: Run automated system tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
```
Comment (smallhive) [inline .github/workflows/system-tests.yml:10]: I thought that we already removed all support branches. Are we planing to handle them? @roman-khimov

Comment (roman-khimov) [reply .github/workflows/system-tests.yml:10]: They're gone. Copy-pasting them is not necessary.

Comment (evgeniiz321) [reply .github/workflows/system-tests.yml]: Removed.

**Comment by roman-khimov** [comment]
It certainly works, but results are rather strange.

**Comment by evgeniiz321** [comment]
For neofs-testcases and neofs-node we use not the latest rest-gw release - v0.7.2. 
So these are the results on the latest version. We should deal with them separately. 
This PR seems to be working as expected.

**Comment by roman-khimov** [comment]
Well, open issues then, please.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**